### PR TITLE
Expose public key from private key

### PR DESCRIPTION
**Pull request by nolasconapoleao**
Added sample code for create_public_key_from_private_key

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -197,6 +198,14 @@ void sr25519_uniform_keypair_from_seed(sr25519_keypair keypair, const sr25519_mi
...
 #+END_SRC
 [[https://github.com/TerenceGe/sr25519-donna/blob/7dd704c0530e7aad50c7ec8e6069725f6124645a/example/src/main.c#L148][example]]
+** create public_key from private_key
+| param         | description                                                          |
+| public_key    | the corresponding public key that signing the message, 32 bytes long |
+| private_key   | the private key of the keypair to sign the message, 64 bytes long    |
+#+BEGIN_SRC C
+void sr25519_public_key_from_private_key(sr25519_public_key public_key, sr25519_secret_key private_key)
+#+END_SRC
+[[https://github.com/appliedblockchain/sr25519-donna-sgx/blob/master/example/src/main.c#L233][example]]
```
Comment (nolasconapoleao) [inline README.org:208]: Normally we point at blobs, but I don't know how to get a hash before committing.

File: src/sr25519.h [hunk 2]
```diff
@@ -38,6 +38,11 @@ typedef struct VrfResult {
     bool is_less;
 } VrfResult;
 
+/**
+ * public_key: the public key of the keypair to sign the message, 32 bytes long
+ * private_key: the secret key of the keypair to sign the message, 64 bytes long
+ */
+void sr25519_public_key_from_private_key(sr25519_public_key public_key, sr25519_secret_key private_key);
```
Comment (nolasconapoleao) [inline src/sr25519.h:45]: Can't understand the warning here
sr25519_public_key_from_private_key

Comment (tgrbrooks) [reply src/sr25519.h:45]: What warning?

**Review by tgrbrooks — APPROVED** [review]
Looks good, thank you!

File: src/sr25519.h (hunk 2 above)
Comment (nolasconapoleao) [reply src/sr25519.h:45]: old-style parameter declarations in prototyped function definition

Comment (tgrbrooks) [reply src/sr25519.h:45]: I don't see that warning either in VSCode or compiling, so it's probably ok to ignore

**Pull request merged**